tools/clang/lib/Sema/SemaHLSL.cpp

```diff
@@ -11882,6 +11913,33 @@ bool hlsl::ShouldSkipNRVO(clang::Sema &sema, clang::QualType returnType,
   return false;
 }
 
+bool hlsl::HasLongVecs(const QualType &qt) {
```

Comment:
nit: https://llvm.org/docs/CodingStandards.html#name-types-functions-variables-and-enumerators-properly

Reply:
As above, this is not part of this PR, but part of the prerequisite #7143. I renamed it inline with your suggestion on that change, which I'm afraid doesn't match the coding standards you cited. https://github.com/microsoft/DirectXShaderCompiler/pull/7143/commits/e010223f74f9c2d96d51349849cd5f5e99542e99#r1960826249